inspirehep/modules/records/cli.py

```diff
@@ -0,0 +1,33 @@
+import click
+
+from .checkers import obtain_references_without_record, create_table_of_references_missing_record
+
+@click.group()
+def ref_wo_rec():
+	"""Commands to obtain references that have no record object"""
+
+
+@ref_wo_rec.command()
+def write_references():
+	click.echo(obtain_references_without_record)
+
+
+@ref_wo_rec.command()
+@ref_wo_rec.argument('file_name',type=str)
+@ref_wo_rec.option('--doi','-d',isflag=True, default=False,
+				   help='Write a file only with references with DOI')
+@ref_wo_rec.option('--arxiv','-a',isflag=True, default=False,
+				   help='Write a file only with references with arxiv id')
+def write_references_on_file():
+	reference_list = obtain_references_without_record
```

Comment:
is this supposed to be a function call? you're missing `()` then. You would have caught this with tests.